### PR TITLE
feat(https): https or ssh git protocol option, closes #5

### LINKED ISSUE #5: Starter gives the incorrect remote link on newly started repo when using https link
I successfully start a new react redux project using rangle-starter.

```
ericjimenez@rngl ~/W/Rangle> rangle-starter react-redux-starter ericjim https://github.com/ericjim/react-playground.git
org: //github.com/ericjim folder: react-playground
Cloning into 'react-playground'...
Saving password to keychain failed
Identity added: /Users/ericjimenez/.ssh/id_rsa ((null))
remote: Counting objects: 87, done.
remote: Compressing objects: 100% (71/71), done.
remote: Total 87 (delta 5), reused 59 (delta 4), pack-reused 0
Receiving objects: 100% (87/87), 23.75 KiB | 0 bytes/s, done.
Resolving deltas: 100% (5/5), done.
Checking connectivity... done.
Initialized empty Git repository in /Users/ericjimenez/Workspace/Rangle/react-playground/.git/
[master (root-commit) 2be5f50] Initial commit from source:
 64 files changed, 1898 insertions(+)
 ....
```

However, my `origin` remote has an invalid link.

```
ericjimenez@rngl ~/W/R/react-playground> git push origin master
ssh: Could not resolve hostname https: nodename nor servname provided, or not known
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Upon further inspection, I see that my `origin` is missing the `//github.com`

```
ericjimenez@rngl ~/W/R/react-playground> git remote --verbose
origin  https:ericjim/react-playground.git (fetch)
origin  https:ericjim/react-playground.git (push)
upstream    https://github.com/ericjim/react-playground.git (fetch)
upstream    https://github.com/ericjim/react-playground.git (push)
```

With that said, since I am using two-step auth, I need to use an ssh key to push to my repo using the starter. So I change the remote to...

```
ericjimenez@rngl ~/W/R/react-playground> git remote rm origin
ericjimenez@rngl ~/W/R/react-playground> git remote add origin git@github.com:ericjim/react-playground.git
ericjimenez@rngl ~/W/R/react-playground> git push origin master
Counting objects: 87, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (75/75), done.
Writing objects: 100% (87/87), 23.54 KiB | 0 bytes/s, done.
Total 87 (delta 5), reused 0 (delta 0)
To git@github.com:ericjim/react-playground.git
 * [new branch]      master -> master
```

And that's how I circumvent the malformed origin link when using the starter.

### PR DESCRIPTION
**Pull request by kosz**
closes #5

### CODE REVIEW

File: src/inquire.js [hunk 1]
```diff
@@ -34,6 +34,16 @@ const TECH_STACK = {
   ]
 };
 
+const USE_HTTPS = {
+  type: 'list',
+  name: 'useHttps',
+  message: 'Please chose a protocol for connecting to github',
+  choices: [
+    { name: 'https: ( https://github.com/yourOrg/yourRep )', value: true },
+    { name: 'ssh: ( git@github.com:yourOrg/yourRepo )', value: false }
```
Comment (JaKXz) [inline src/inquire.js]: Is SSH still the default choice? That'd be preferable

File: src/execute.js [hunk 2]
```diff
@@ -5,17 +5,19 @@ const log = require('./log');
...
+    buildGitUrl(config.repoForkOrg, config.repoName, config.useHttps) : false;
 
   clone(config.techStack, gitFork, gitOrigin, config.repoName)
     .catch(log);
 };
 
-function buildGitUrl(orgName, projectName) {
-  return `git@github.org:${orgName}/${projectName}.git`;
-}
+const HTTPS = 'https://github.org/';
```
Comment (JaKXz) [inline src/execute.js]: I'm guessing you've tested this, but isn't it github.com?

File: src/inquire.js (hunk 1 above)
Comment (kosz) [inline src/inquire.js]: @JaKXz thoughts on this ? https://github.com/rangle/rangle-starter/issues/5#issuecomment-200419688

File: src/execute.js (hunk 2 above)
Comment (kosz) [inline src/execute.js]: good catch, was just testing output

File: src/execute.js [hunk 3]
```diff
@@ -5,17 +5,19 @@ const log = require('./log');
...
+    buildGitUrl(config.repoForkOrg, config.repoName, config.useHttps) : false;
 
   clone(config.techStack, gitFork, gitOrigin, config.repoName)
     .catch(log);
 };
 
-function buildGitUrl(orgName, projectName) {
-  return `git@github.org:${orgName}/${projectName}.git`;
-}
+const HTTPS = 'https://github.com/';
```
Comment (kosz) [inline src/execute.js:16]: so what would take to support bitbucket here ? :)

Comment (SethDavenport) [inline src/execute.js:16]: Maybe another choice in inquirer for the SSH/HTTPS section - could even be "other: (enter url)" or something.  But we can do that later.

**Comment by SethDavenport** [comment]
:shipit:

**Pull request merged**